operations/commit_queue.go

```diff
@@ -0,0 +1,93 @@
+package operations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen/rest/client"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+func CommitQueue() cli.Command {
+	const (
+		listFlagName   = "list"
+		deleteFlagName = "delete"
+		itemFlagName   = "item"
+	)
+
+	return cli.Command{
+		Name:  "commit-queue",
+		Usage: "interact with the commit queue",
+		Flags: addProjectFlag(
```

Comment:
Shouldn't list and delete be subcommands, not flags? And I believe the item should be an arg to delete

Reply:
that's much more elegant.
Done. 
(though with delete as a subcommand I left in the item flag: I wasn't happy about it being an unnamed argument)